hwsc-file-transaction-svc.proto

```diff
@@ -2,26 +2,28 @@ syntax = "proto3";
 
 package hwscFileTransactionSvc;
 
-// Service: define the methods that gRPC server can expose to the client
 service FileTransactionService{
     rpc GetStatus (FileTransactionRequest) returns (FileTransactionResponse){}
     rpc UploadFile (stream Chunk) returns (FileTransactionResponse){}
-    rpc DownloadFile (FileTransactionRequest) returns (stream Chunk){}
+    rpc DownloadZippedFiles (FileTransactionRequest) returns (stream Chunk){}
 }
 
 message FileTransactionRequest{
     string message = 1;
     string name = 2;
-    string url = 3;
+    repeated string urls = 3;
 }
 
 message FileTransactionResponse{
     oneof status {uint32 code = 1;}
     string message = 2;
     string url = 3;
-    int64 length = 4;
 }
 
 message Chunk{
-    bytes buffer = 1;
-}
\ No newline at end of file
+    oneof test_oneof{
+        bytes buffer = 1;
+        string fileName = 2;
```

Comment:
field name should be underscore_seperated:
`fileName` should be `file_name`

https://developers.google.com/protocol-buffers/docs/style#message-and-field-names